packages/calcite-components/src/components/navigation-user/navigation-user.e2e.ts

```diff
@@ -39,4 +41,40 @@ describe("calcite-navigation-user", () => {
   describe("is focusable", () => {
     focusable("calcite-navigation-user");
   });
+
+  describe("theme", () => {
+    const navigationUserHtml = html`
+      <calcite-navigation-user full-name="Walt McChipson" username="waltChip"> </calcite-navigation-user>
+    `;
+
+    describe("default", () => {
+      const tokens: ComponentTestTokens = {
+        "--calcite-navigation-user-avatar-corner-radius": {
+          shadowSelector: `.${CSS.button} >>> calcite-avatar`,
```

Comment:
Is it suggested to query the nested component rather than class and assert it tokens?

Example:
```
 "--calcite-navigation-user-avatar-text-color": {
          shadowSelector: `calcite-avatar`,
          targetProp: "--calcite-avatar-text-color",
        },
```
Vs        
        
```
 "--calcite-navigation-user-avatar-text-color": {
          shadowSelector: `.button`,
          targetProp: "--calcite-avatar-text-color",
        },
```

To support the second option the nested component should have the css class selector 
```
.button {

calcite-avatar {
//tokens
}
}